index.html

```diff
@@ -0,0 +1,157 @@
+
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Mobile Version</title>
+    <script src="https://kit.fontawesome.com/2af886883b.js" crossorigin="anonymous"></script>
+</head>
+<link rel="stylesheet" href="style.css">
+<body>
+    <section>
+        <div class="image1">
+        <div class="img-1">
+        <img  src="assets/img/Snapshoot Portfolio (1).jpg">
```

Comment:
- Kindly, check the image path, the image is broken. 
- You can remove the whitespace between the image path and make it look like this 👉
![img](https://user-images.githubusercontent.com/73854999/198697218-04443108-9b2d-4dde-a019-77318c7aaffe.PNG)
- **Note:** You will have to rename it also. 😊
- You can rename it to any name of yours, just make sure there is no space between them. 
- This is what you have presently...👇
![no-img](https://user-images.githubusercontent.com/73854999/198696837-ac1ffdea-7207-49b3-8e90-c87094de4650.PNG)